include/braincloud/BrainCloudTournament.h

```diff
@@ -55,6 +91,18 @@ namespace BrainCloud
 		 */
 		void joinTournament(const char* in_leaderboardId, const char* in_tournamentCode, int64_t in_initialScore, IServerCallback * in_callback = NULL);
 
+		 /**
+		 * Removes player from division instance
+		 * Also removes division instance from player's division list
+		 *
+		 * Service Name - tournament
+		 * Service Operation - LEAVE_DIVISION_INSTANCE
+		 *
+		 * @param in_leaderboardId The leaderboard for the tournament
+		 * @param in_callback The method to be invoked when the server response is received
+		 */
+		void leaveDivisionInstance(const char* in_leaderboardId, IServerCallback * in_callback = NULL);
```

Comment:
not a big deal, but just for consistency:
IServerCallback * in_callback
IServerCallback* in_callback